CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 
 > Note: due to an issue with deployment, this was released as v1.0.3.2
 
+* Python 2 support is removed ([#31])
+* Python 3.8 support is added
```

Comment:
#31 is not in the current release 1.0.3.2 -- maybe just move this to 1.0.4 and I'll release another version to include all changes?

Reply:
Ah of course. My mistake. Fixing now.